### ISSUE TITLE
Feature request: add locale

### ISSUE THREAD

**Issue by EKE00372** [obble/modui_classic#10]
Would you mind to add locales? I can support zhCN/zhTW.

**Comment by obble** [comment]
Yeah - i'd be keen to add this support! Let me collect together text strings that would need translating in the next few days.

I guess chat text might be complex for finding and matching different sentence structures across different locales, but lets see.

**Comment by EKE00372** [comment]
ah yes that's a complex work.....even grammar different. or just options at first?

**Comment by WrobotProfiles** [comment]
Hey Obble, are you getting this Email ?

**Comment by obble** [comment]
Hey there, so I added a file for locales:

https://github.com/obble/modui_classic/blob/master/core/locale.lua

The formatting in this table layout follows the rule
L['**Text string to translate**'] = _'translation'_

Feel free to duplicate this for the two languages you're able to support. You can either post here in a pastebin or the like, or make a pull request.

**Comment by EKE00372** [comment]
Working on it, soon(tm)

**Comment by EKE00372** [comment]
I'm planing translate channel (such like '|Hchannel:Guild|hG|h. %s:\32') and other global strings, maybe will add them after locale get apply, because i need a little.....FEELING on typesetting when translate them >.>

**Comment by icemansparks** [comment]
I created PR #105 for german translation.
